theme/redbox.js

```diff
@@ -1,38 +1,101 @@
-  // Append the new div to the parent element
-  parentElement.insertBefore(newDiv, parentElement.firstChild);
-  // Create the button element
-  var hideShowButton = document.createElement("button");
-  hideShowButton.innerHTML = '<i class="fa fa-square-o"></i>';
-  hideShowButton.className = "icon-button";
-  hideShowButton.type = "button";
-  hideShowButton.title =
-    "Outline the area that fits on one screen while teaching the course.";
-  hideShowButton.id = "Dev";
-  var navbarButtons = document.getElementsByClassName("left-buttons");
-  navbarButtons[0].insertBefore(hideShowButton, navbarButtons.firstChild);
-  //Default hiding the redbox
-  document.getElementById("aspect-ratio-helper").style.display = "none";
-  //Add Event listener to button to perform on click action.
-  hideShowButton.addEventListener("click", function () {
-    if (
-      document.getElementById("aspect-ratio-helper").style.display === "none"
-    ) {
-      document.getElementById("aspect-ratio-helper").style.display = "block";
-      hideShowButton.innerHTML = '<i class="fa fa-square"></i>';
-    } else {
-      document.getElementById("aspect-ratio-helper").style.display = "none";
-      hideShowButton.innerHTML = '<i class="fa fa-square-o"></i>';
-    }
-  });
+(function handleInstructor(params) {
+  function handleInstructorMenu() {
+    let leftButtons = document.getElementsByClassName("left-buttons")[0];
+    let instructorMenu = document.createElement("button");
+    let instructorMenuList = document.createElement("ul");
+    let redBoxItem = document.createElement("li");
+    let redBoxButton = document.createElement("button");
+    let playgroundStateItem = document.createElement("li");
+    let playgroundStateButton = document.createElement("button");
+
+    leftButtons.insertBefore(instructorMenu, leftButtons.lastChild);
+    leftButtons.insertBefore(instructorMenuList, leftButtons.lastChild);
+    instructorMenuList.insertBefore(redBoxItem, instructorMenuList.lastChild);
+    instructorMenuList.insertBefore(
+      playgroundStateItem,
+      instructorMenuList.lastChild
+    );
+    redBoxItem.insertBefore(redBoxButton, redBoxItem.lastChild);
+    playgroundStateItem.insertBefore(
+      playgroundStateButton,
+      playgroundStateItem.lastChild
+    );
+
+    instructorMenu.innerHTML =
+      '<i class="fa fa-ellipsis-v" aria-hidden="true"></i>';
+    redBoxButton.innerHTML = "redbox";
+    playgroundStateButton.innerHTML = "reset code";
+
+    instructorMenu.className = "icon-button";
+    instructorMenuList.className = "theme-popup";
+    redBoxButton.className = "theme";
+    playgroundStateButton.className = "theme";
+    instructorMenuList.style.display = "none";
+
+    instructorMenuList.role = "menu";
+    redBoxItem.role = "none";
+    playgroundStateItem.role = "none";
+    redBoxButton.role = "menuitem";
+    playgroundStateButton.role = "menuitem";
+
+    redBoxButton.id = "redbox";
+
+    instructorMenuList.style.marginLeft = "55px";
```

Comment:
Minor detail: I think this would be easier to read if each item was grouped together, and all its properties set in the same place.